### PR TITLE
prepare Shapiro/more htests in model_info

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

**Comment by strengejacke** [comment]
@mattansb any idea how to name these, r which family?
See https://github.com/easystats/insight/pull/638/files#diff-82d553d182838f9d82da9937dee59f8a4d6e588f2d84c2730caf4b6a2b54bd25

**Comment by mattansb** [comment]
I guess Bartlett's test is closest to Levene's test?
Maybe we can add a general "is_variance_test"? Which both would kind of be?

**Comment by strengejacke** [comment]
ok, sounds good.

**Comment by strengejacke** [comment]
what about "family"? Is there any particular "family" at all? (except maybe gaussian for shapiro)

**Comment by strengejacke** [comment]
@easystats/core-team This PR should provide more information in `model_info()` to be able to deal with https://github.com/easystats/report/issues/256. Any suggestions, or approvals? We've been asked to update _insight_ on CRAN within the next two weeks, so we could possibly include this PR in the next submission.

**Comment by rempsyc** [comment]
I can't see a reprex to comment on the output but I approve the concept!

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # insight 0.18.4
 
+## Changes to functions
+
+* `model_info()` now includes information for `htest` objects from
+  `shapiro.test()` and `bartlett.test()` (will return `$is_varianctest = TRUE`).
```
Comment (DominiqueMakowski) [inline NEWS.md:6]:
```suggestion
  `shapiro.test()` and `bartlett.test()` (will return `$is_variancetest = TRUE`).
```

**Comment by DominiqueMakowski** [comment]
looks good

**Comment by strengejacke** [comment]
> I can't see a reprex to comment on the output but I approve of the concept!

It's more about the naming. What should `model_info()` return for Shapiro and Bartlett tests? Currently, we have `$is_variancetest`. We need a distinction from other htests for _report_ (see https://github.com/easystats/report/issues/256).

**Comment by strengejacke** [comment]
ok, here is "variance" mentioned in conjunction with "shapiro-wilk-test", so it's probably ok:
https://www.jstor.org/stable/2333709

**Pull request merged**